airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -372,32 +370,43 @@ definitions:
     properties:
       type:
         type: string
-        enum: [DatetimeStreamSlicer]
+        enum: [DatetimeBasedCursor]
       cursor_field:
+        description: The location of the value on a record that will be used as a bookmark during sync
         type: string
       datetime_format:
+        description: The format of the datetime
         type: string
       cursor_granularity:
+        description: Smallest increment the datetime_format has (ISO 8601 duration) that is used to ensure the start of a slice does not overlap with the end of the previous one
         type: string
       end_datetime:
+        description: The datetime that determines the last record that should be synced
         anyOf:
           - type: string
           - "$ref": "#/definitions/MinMaxDatetime"
       start_datetime:
+        description: The datetime that determines the earliest record that should be synced
         anyOf:
           - type: string
           - "$ref": "#/definitions/MinMaxDatetime"
       step:
+        description: The size of the time window (ISO8601 duration)
         type: string
       end_time_option:
+        description: Request option for end time
         "$ref": "#/definitions/RequestOption"
       lookback_window:
+        description: How many days before start_datetime to read data for (ISO8601 duration)
         type: string
       start_time_option:
+        description: Request option for start time
         "$ref": "#/definitions/RequestOption"
-      stream_state_field_end:
+      partition_field_end:
```

Comment:
nit: Can you move partition_field_end and partition_field_start above the start_time_option? The overall organization was to alphabetize the required fields, then alphabetize the non required fields